app/services/vot_parser.rb

```diff
@@ -0,0 +1,68 @@
+class VotParser
+  class VotParseException < StandardError; end
+
+  VotParserResult = Data.define(:identity_proofing, :credential_usage)
```

Comment:
since these classes are already nested under the `VotParser` namespace I think we could shorten their names
```suggestion
  class ParseException < StandardError; end

  Result = Data.define(:identity_proofing, :credential_usage)
```